docs/data/material/customization/palette/palette.md

```diff
@@ -108,16 +108,41 @@ const theme = createTheme({
 });
 ```
 
-As in the example above, if the palette color contains custom colors using any of the
-"main", "light", "dark" or "contrastText" keys, these map as follows:
+For the default palette colors (the ones listed in the [Default values section](#default-values) above), only the `main` token is required. If the other tokens are not provided their value is calculated as follows:
 
-- If the "dark" and / or "light" keys are omitted, their value(s) will be calculated from "main",
-  according to the "tonalOffset" value.
-- If "contrastText" is omitted, its value will be calculated to contrast with "main",
-  according to the "contrastThreshold" value.
+- The `light` and `dark` tokens are calculated according to the `tonalOffset` value with the `main` color as a starting point.
+- The `contrastText` token is calculated according to the `contrastThreshold` value to contrast with the `main` color.
 
-Both the "tonalOffset" and "contrastThreshold" values may be customized as needed.
-The "tonalOffset" value can either be a number between 0 and 1, which will apply to both light and dark variants, or an object with light and dark variants specified by the following TypeScript type:
+**Values for custom palette colors won't be automatically calculated** like for the default palette colors.
```

Comment:
We could add a [warning callout](https://www.notion.so/mui-org/Callouts-and-their-usage-in-the-docs-7e709a02c72142cd8b1a0829861435c5) here since this fits the definition of something a dev should keep in mind to avoid potential problems.

```suggestion
:::warning
Unlike default palette colors, values for custom palette colors are *not* automatically calculated.
:::
```